### DISCUSSION TITLE
RegisterWindowCreatingHandler is not triggered when opening a link

### DISCUSSION THREAD

**Discussion by gerwim** [tryphotino/photino.NET#147] [Q&A]
I was under the impression the handler registered to `RegisterWindowCreatingHandler` would be triggered upon clicking a link (which opens a new window) -- but that is not the case.

What I'd like: to open the user installed browser upon clicking a link instead of opening it in the Photino window.

**Comment by MikeYeager** [comment]
@gerwim Clicking a link in in Photino doesn't create a new window, so RegisterWindowCreatingHandler won't help here. The issue is similar to browsing a page in Chrome and having a link open a new page in Edge. Browsers don't typically open other browsers. What you can do is handle a message in .NET and have .NET spawn a new process. That would open the link in the OS's default browser.
